platform/linux-generic/include-abi/odp/api/abi/dma_types.h

```diff
@@ -21,7 +21,7 @@ typedef ODP_HANDLE_T(odp_dma_t);
 
 #define ODP_DMA_INVALID _odp_cast_scalar(odp_dma_t, 0)
 
-typedef uint32_t odp_dma_transfer_id_t;
+typedef uint64_t odp_dma_transfer_id_t;
```

Comment:
Changed in so that `linux-gen` implementation always just handles 32 bit transfer IDs, the in-flight transfer count likely never getting anywhere close to the max value so should be fine. In v2.